src/apps/vpn/cmake/windows.cmake

```diff
@@ -4,25 +4,25 @@
 
 add_definitions(-DWIN32_LEAN_AND_MEAN)
 
-set_target_properties(mozillavpn PROPERTIES
+set_target_properties(${MAIN_TARGET} PROPERTIES
```

Comment:
This property values is what you want to change. Not the rest.

Reply:
Pleaes see my previous comment. I have not had success on changing the XCode scheme name just by changing a target property.